### PR TITLE
Lint all JavaScript files

### PR DESCRIPTION
**Pull request by lpinca**
This patch brings the following changes:
- Update `standard` to version `6.x`.
- Run the linter on all JavaScript files.
- Clean up the scripts to pull events data.
- Fix existing lint errors.

### CODE REVIEW

**Comment by ghost** [comment]
[Travis build passed](https://travis-ci.org/nodejs/nodejs.org/builds/118480244) :+1:

File: events/pull-nodeschool.js [hunk 1]
```diff
@@ -1,16 +1,25 @@
...
+  formatter: null
+})
+
+request({
+  headers: { 'user-agent': `${pkg.name}/${pkg.version}` },
+  url: 'http://nodeschool.io/chapters/list.json',
+  json: true
+}, (err, resp, list) => {
+  if (err || resp.statusCode !== 200) {
+    throw (err || new Error('response not 200' + resp.statusCode))
```
Comment (phillipj) [inline events/pull-nodeschool.js]: Some nitpicking for you; the error message should probably have a space added after `200`.

File: package.json [hunk 2]
```diff
@@ -9,7 +9,7 @@
     "load-versions": "node scripts/load-versions.js",
     "start": "npm run serve",
     "test": "npm run test:lint && npm run test:unit && npm run test:smoke",
-    "test:lint": "standard *.js scripts/*.js scripts/**/*.js tests/*.js tests/**/*.js",
+    "test:lint": "standard",
```
Comment (phillipj) [inline package.json:12]: :+1:

File: events/pull-nodeschool.js (hunk 1 above)
Comment (lpinca) [inline events/pull-nodeschool.js]: This was copy/pasted but it definitely makes sense. Gonna fix it in a bit.
The same thing need to be fixed also in the other file.

File: events/pull-meetup.js [hunk 3]
```diff
@@ -37,22 +36,36 @@ function finish (events) {
...
+      if (title.includes('nodeschool')) return
+      if (title.includes('mongodb') && title.includes('node')) return
+      if (title.includes('find a tech job') && title.includes('node')) return
+
+      results.push(result)
     })
+
     if (body.meta.next) {
-      _go(body.meta.next)
+      pull(Object.assign({}, defaults, { url: body.meta.next }))
```
Comment (phillipj) [inline events/pull-meetup.js]: Any reason you didn't go for `Object.assign({ url: body.meta.next }, defaults)`?

Comment (lpinca) [inline events/pull-meetup.js]: > Sometimes my mind plays tricks on me.

:smile:

**Comment by lpinca** [comment]
@phillipj PTAL.

**Comment by phillipj** [comment]
Very nice :+1: Even seems to work locally, so lets :shipit: Maybe some day we'll even have tests on this...

**Pull request merged**